result/isislsp.yaml

```diff
@@ -50,9 +38,14 @@ components:
       properties:
         lsp_id:
           description: >-
-            LSP ID of the LSP, e.g. '6400000000010000'.
+            LSP ID of the LSP, in the format, e.g. '640000000001-00-00'.
+            LSP ID is consist of the System ID of a neighbor, the Pseudonode ID, and the LSP number of the LSPs.
```

Comment:
LSP ID is consist of --> LSP ID consists of
and the LSP number of the LSPs --> and the LSP number.